test/CMakeLists.txt

```diff
@@ -1,9 +1,24 @@
 cmake_minimum_required(VERSION 3.7)
 
 enable_testing()
-find_package(GTest REQUIRED)
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  # Specify the commit you depend on and update it regularly.
+  URL https://github.com/google/googletest/archive/refs/tags/v1.13.0.zip
+  # FIND_PACKAGE_ARGS NAMES GTest
+)
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+# Content googletest already populated in FetchContent_Populate(googletest)
+# find_package(GTest REQUIRED)
 
-find_library(LIBTINS_LIBRARIES tins)
+message(STATUS "googletest populate: " ${googletest_SOURCE_DIR} " " ${googletest_BINARY_DIR})
```

Comment:
This message could be more meaningful. What does it mean?
Also, if I just want to build the library, I don't need to see this - instead of a `STATUS` message this could be `DEBUG`.

Reply:
Debug message again, not needed anymore